pypeit/core/transform.py

```diff
@@ -0,0 +1,140 @@
+"""
+Provide basic coordinate tranformation functions.
+
+.. include common links, assuming primary doc root is up one directory
+.. include:: ../include/links.rst
+"""
+
+from IPython import embed
+
+import numpy as np
```

Comment:
Put these routines in the mosaic module?

Reply:
I prefer them to be separate.